webpack/assets/javascripts/react_app/redux/API/APIMiddleware.js

```diff
@@ -8,4 +8,5 @@ export const APIMiddleware = store => next => action => {
   } else {
     next(action);
   }
+  return next(action);
```

Comment:
I think the request for the return statement to be placed in each branch of the above  if statement, as this line duplicates the 'else' condition immediately above

Reply:
Exactly what I was in the process of typing.

Reply:
oh right, missed that.

Reply:
I think just running next(action) has a different result from returning it, so I'll return the results of both if branches like @ezr-ondrej suggested.